public/app/features/explore/LogsVolumePanelList.tsx

```diff
@@ -0,0 +1,123 @@
+import { css } from '@emotion/css';
+import { groupBy } from 'lodash';
+import React, { useMemo } from 'react';
+
+import {
+  AbsoluteTimeRange,
+  DataFrame,
+  DataQueryResponse,
+  EventBus,
+  GrafanaTheme2,
+  isLogsVolumeLimited,
+  LoadingState,
+  SplitOpen,
+  TimeZone,
+} from '@grafana/data';
+import { Button, InlineField, useStyles2 } from '@grafana/ui';
+
+import { LogsVolumePanel } from './LogsVolumePanel';
+import { SupplementaryResultError } from './SupplementaryResultError';
+
+type Props = {
+  logsVolumeData: DataQueryResponse | undefined;
+  absoluteRange: AbsoluteTimeRange;
+  timeZone: TimeZone;
+  splitOpen: SplitOpen;
+  width: number;
+  onUpdateTimeRange: (timeRange: AbsoluteTimeRange) => void;
+  onLoadLogsVolume: () => void;
+  onHiddenSeriesChanged: (hiddenSeries: string[]) => void;
+  eventBus: EventBus;
+};
+
+export const LogsVolumePanelList = ({
+  logsVolumeData,
+  absoluteRange,
+  onUpdateTimeRange,
+  width,
+  onLoadLogsVolume,
+  onHiddenSeriesChanged,
+  eventBus,
+  splitOpen,
+  timeZone,
+}: Props) => {
+  const logVolumes = useMemo(
+    () => groupBy(logsVolumeData?.data || [], 'meta.custom.sourceQuery.refId'),
+    [logsVolumeData]
+  );
+
+  const styles = useStyles2(getStyles);
+
+  const numberOfLogVolumes = Object.keys(logVolumes).length;
+
+  const containsZoomed = Object.values(logVolumes).some((data: DataFrame[]) => {
+    const zoomRatio = logsLevelZoomRatio(data, absoluteRange);
+    return !isLogsVolumeLimited(data) && zoomRatio && zoomRatio < 1;
+  });
+
+  let zoomedInfo;
+  if (containsZoomed) {
+    zoomedInfo = (
+      <InlineField label="Reload log volume" transparent>
+        <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} id="reload-volume" />
+      </InlineField>
+    );
+  }
+
+  if (logsVolumeData?.state === LoadingState.Loading) {
+    return <span>Loading...</span>;
+  }
+  if (logsVolumeData?.error !== undefined) {
+    return <SupplementaryResultError error={logsVolumeData.error} title="Failed to load log volume for this query" />;
+  }
+  return (
+    <div className={styles.listContainer}>
+      {Object.keys(logVolumes).map((name, index) => {
+        const logsVolumeData = { data: logVolumes[name] };
+        return (
+          <LogsVolumePanel
+            key={index}
+            absoluteRange={absoluteRange}
+            width={width}
+            logsVolumeData={logsVolumeData}
+            onUpdateTimeRange={onUpdateTimeRange}
+            timeZone={timeZone}
+            splitOpen={splitOpen}
+            onLoadLogsVolume={onLoadLogsVolume}
+            // TODO: Support filtering level from multiple log levels
+            onHiddenSeriesChanged={numberOfLogVolumes > 1 ? () => {} : onHiddenSeriesChanged}
+            eventBus={eventBus}
+          />
+        );
+      })}
+      <div className={styles.extraInfoContainer}>{zoomedInfo}</div>
```

Comment:
kinda nit:
```suggestion
      {containsZoomed && (
        <div className={styles.extraInfoContainer}>
          <InlineField label="Reload log volume" transparent>
            <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} id="reload-volume" />
          </InlineField>
        </div>
      )}
```

and removing the block at https://github.com/grafana/grafana/pull/63036/files#diff-02d91743e836fe991966b196cd2c26c3571b756af6a7032e4142e3b1f0c53926R58-R65, so we also avoid to render that extra empty div